plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -301,6 +307,47 @@ const config: ControlPanelConfig = {
           },
         ],
         // eslint-disable-next-line react/jsx-key
+        [<h1 className="section-header">{t('X Axis')}</h1>],
+        ['x_axis_time_format'],
```

Comment:
<img src="https://user-images.githubusercontent.com/335541/104712205-2003a800-56d7-11eb-9c5c-491f4f66234a.png" width="300">

This will indeed show two copies of the same control when you `npm link` both `chart-controls` and the plugin.